examples/ChatQnA/deploy/k8s_getting_started.md

```diff
@@ -0,0 +1,90 @@
+- **Using Manifests**: This involves deploying directly using Kubernetes manifest files without the GenAI Microservices Connector (GMC).
+- **Using Helm Charts**: Facilitates deployment through Helm, which manages Kubernetes applications through packages of pre-configured Kubernetes resources.
+
+This guide will provide detailed instructions on using these resources. If you're already familiar with Kubernetes, feel free to skip ahead to [Helm Deployment](./k8s_helm.md)
+
+### Kubernetes Cluster and Development Environment
+
+**Setting Up the Kubernetes Cluster:** Before beginning deployment for the ChatQnA application, ensure that a Kubernetes cluster is ready. For guidance on setting up your Kubernetes cluster, please refer to the comprehensive setup instructions available on the [Opea Project deployment guide](https://opea-project.github.io/latest/deploy/index.html).
+
+**Development Pre-requisites:** To prepare for the deployment, familiarize yourself with the necessary development tools and configurations by visiting the [GenAI Infrastructure development page](https://opea-project.github.io/latest/GenAIInfra/DEVELOPMENT.html). This page covers all the essential tools and settings needed for effective development within the Kubernetes environment.
+
+
+**Understanding Kubernetes Deployment Tools and Resources:**
+
+- **kubectl**: This command-line tool allows you to deploy applications, inspect and manage cluster resources, and view logs. For instance, `kubectl apply -f chatqna.yaml` would be used to deploy resources defined in a manifest file.
+    
+- **Pods**: Pods are the smallest deployable units created and managed by Kubernetes. A pod typically encapsulates one or more containers where your application runs.
+
+**Verifying Kubernetes Cluster Access with kubectl**
+```bash
+kubectl get nodes
+```
+
+This command lists all the nodes in the cluster, verifying that `kubectl` is correctly configured and has the necessary permissions to interact with the cluster.
+
+Some commonly used kubectl commands and their functions that will help deploy ChatQnA successfully:
+|Command                          |Function                     |
+|-------------------------------  |-----------------------------|
+|`kubectl describe pod <pod-name>` | Provides detailed information about a specific pod, including its            current state, recent events, and configuration details.                            |
+|`kubectl delete deployments --all` |             Deletes all deployments in the current namespace, which effectively removes all the managed pods and associated resources.                |
+|`kubectl get pods -o wide`         |              Retrieves a detailed list of all pods in the current namespace, including additional information like IP addresses and the nodes they are running on.               |
```

Comment:
Use of "current" in all of these is wrong.

When namespace is not specified, removal is done from "default" namespace, not "current" one, whatever that is.

=> I think all examples should include namespace option,.

Reply:
Moved the `Create and Set Namespace` above this section to indicate what `current` namespace means.